rtran-maven/src/main/scala/com/ebay/rtran/maven/util/MavenUtil.scala

```diff
@@ -96,6 +96,12 @@ object MavenUtil {
     val dependencyFilter = new ExclusionsDependencyFilter(
       dependency.getExclusions.map(e => s"${e.getGroupId}:${e.getArtifactId}")
     )
+
+    def recoverFromResolutionException: PartialFunction[Throwable, util.List[Artifact]] = {
```

Comment:
the codacy issue doesn't make sense for me:
Why do I need to provide a default case for a PartialFunction like below:
```scala
    def recoverFromResolutionException: PartialFunction[Throwable, util.List[Artifact]] = {
      case e: DependencyResolutionException =>
        e.getResult.getArtifactResults.flatMap(r => Option(r.getArtifact))
    }
```